runfile/physics.py

```diff
@@ -130,6 +130,18 @@ def compare_data(exp_data, ref_data):
         from config import *
         import microphys.drivers.gfdl_cloud_microphys_gt4py as phy
 
+    elif args.parameterization == "get_prs_fv3":
+        PRS_DIR = "../get_prs_fv3/python/"
```

Comment:
Make sure to update here if you decide to change the folder name

Reply:
Done!